sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

```diff
@@ -260,3 +295,74 @@ sechub -f mark-fp.json unmarkFalsePositives
 ----
 with a corresponding `mark-fp.json` - see <<section-client-false-positives-unmark,Unmark false positives JSON format>> 
 
+[[section-client-interactiveMarkFalsePositives-mark]]
+===== interactiveMarkFalsePositives
+[source, shell]
+```
+SECHUB_APITOKEN=7536a8c4aa82407da7e06bdbEXAMPLE
+sechub interactiveMarkFalsePositives
+
+/  ___|         | | | |     | |    
+\ `--.  ___  ___| |_| |_   _| |__  
+ `--. \/ _ \/ __|  _  | | | | '_ \ 
+/\__/ /  __/ (__| | | | |_| | |_) |
+\____/ \___|\___\_| |_/\__,_|_.__/ Client Version 0.20.0-b78
+
+Using latest report file "sechub_report_0a84899f-1305-41e0-a2d5-e7c31d6185fd.json".
+------------------------------------------------------------------
+1: Side Channel Data Leakage, severity: HIGH
+src/main/java/com/example/testapp/invoice/application/InvoiceApplicationService.java, line 651 column 17
+        Invoice creditNote = getInvoice(invoiceId).orElseThrow(InvoiceNotFoundException::new);
+------------------------------------------------------------------
+Add this as false positive? ('y' Yes, 'n' No, 's' Skip following findings, 'c' Cancel) y
```

Comment:
Here the `s` option has a different explanation, compared to line https://github.com/Daimler/sechub/pull/363/files#diff-72bc17022d074618aaa8cacc90dbe7d5R356. Please make it consistent.

Reply:
Good point :+1: 
Fixed